scripts/samplesheet_generator.py

```diff
@@ -15,7 +15,7 @@
 DESC = """EPP used to create samplesheets for Illumina sequencing platforms"""
 
 # Pre-compile regexes in global scope:
-IDX_PAT = re.compile("([ATCG]{4,})-?([ATCG]*)")
+IDX_PAT = re.compile("([ATCGN]{5,})-?([ATCG]*)")
```

Comment:
Should work 👍 
Maybe something like `([ATCG]{4,})N*-?([ATCG]*)` if you want to be more specific.

Reply:
Hi @jgruselius, thank you for the suggestion! I tested and the specific expression should be `([ATCG]{4,}N*)-?([ATCG]*)`. The code has been updated.

Reply:
If you want the N's in the capture group along with index 1, then yes.

Reply:
I don't think that we do want the N's in the capture group do we..?

Reply:
I guess we need the N's to set correct cycle numbers in the masks?